source/retryable-writes/tests/updateMany.yml

```diff
@@ -1,12 +1,16 @@
+runOn:
+    -
+        minServerVersion: "3.6"
+        topology: ["replicaset", "sharded"]
+
 data:
     - { _id: 1, x: 11 }
     - { _id: 2, x: 22 }
 
-minServerVersion: '3.6'
-
 tests:
     -
         description: "UpdateMany ignores retryWrites"
```

Comment:
This is unrelated to this review but shouldn't this file also have a test that enables the fail point and confirms that `UpdateMany` is not retried after a network error?

Reply:
Opened: https://jira.mongodb.org/browse/SPEC-1241